_appmap/configuration.py

```diff
@@ -269,6 +274,40 @@ def _load_functions(self):
 
         self.package_functions.update(modules)
 
+    def _drop_malformed_package_paths(self):
+        invalid_items = []
+        for item in self._config["packages"]:
+            # it can be a "dist" entry
+            if not "path" in item:
+                continue
+
+            path = item.get("path")
+            if path is None:
+                logger.warning("Missing path value in configuration file.")
+                invalid_items.append(item)
+                continue
+
+            if not self._check_path_value(path):
+                has_separator = isinstance(path, str) and ('/' in path or '\\' in path)
+                logger.warning(
+                    f"Malformed path value '{path}' in configuration file. "
+                    "Path entries must be module names"
+                    f"{' not directory paths' if has_separator else ''}."
+                )
+                invalid_items.append(item)
+                continue
+
+        if len(invalid_items) > 0:
```

Comment:
What's going to happen if this eliminates all the path entries?

Reply:
I think, no package will be instrumented, unless there are some `dist` entries. We reach this state with at least one warning.

Thinking again, should we check if:
 1. `packages` is a yaml sequence and not a scalar
 2. `packages` is empty
 3. Every item has `path` XOR `dist` key

Reply:
No, let's not expand the scope of this beyond checking that `path`s are modules.

However, please add a test that has only invalid `path`s, and confirm that the resulting `Config` has a `package` with an empty list of `paths`.

Reply:
Done